demo/main.js

```diff
@@ -122,13 +122,30 @@ class Demo extends Component {
           const myjszip = new JSZip();
           const zip = await myjszip.loadAsync(input.files[0]);
           const files = Object.keys(zip.files);
-          const contents = await Promise.all(files.map((file) => zip.files[file].async("text")));
+          const images = [];
+          const contents = await Promise.all(
+            files.map((file) => {
+              if (file.includes("media/image")) {
+                images.push(file);
+                return zip.files[file].async("blob");
+              }
+              return zip.files[file].async("text");
+            })
+          );
           const inputFiles = {};
           for (let i = 0; i < contents.length; i++) {
             inputFiles[files[i]] = contents[i];
           }
           this.leaveCollaborativeSession();
           await fetch("http://localhost:9000/clear");
+          for (let i = 0; i < images.length; i++) {
+            const blob = inputFiles[images[i]];
+            const path = await this.fileStore.upload(
+              new File([blob], images[i].split("/").reverse()[0])
+            );
+            const size = await this.getImageSize(path);
+            inputFiles[images[i]] = [path, size.width, size.height];
```

Comment:
@hokolomopo It was because of the module in Enterprise (demo is just mock what enterprise did). I couldn't use POJO at the first time when I tried, for unknown reason. But now it seems work again 😅 so I'll go for POJO too :)